src/api/chainHead_unstable_follow.md

```diff
@@ -129,6 +129,148 @@ All items in `finalizedBlockHashes` and `prunedBlockHashes` are guaranteed to ha
+    "value": [...]
+}
+```
+
+`operationId` is a string returned by `chainHead_unstable_body`.
+
+The `operation-body-done` event indicates that an operation started with `chainHead_unstable_body` was successful.
+
+`value` is an array of strings containing the hexadecimal-encoded SCALE-encoded extrinsics found in the block.
+
+**Note**: Note that the order of extrinsics is important. Extrinsics in the chain are uniquely identified by a `(blockHash, index)` tuple.
+
+No more event will be generated with this `operationId`.
+
+### operation-call-done
+
+```json
+{
+    "event": "done",
+    "operationId": ...,
+    "output": "0x0000000..."
+}
+```
+
+`operationId` is a string returned by`chainHead_unstable_call`.
+
+The `operation-call-done` event indicates that an operation started with `chainHead_unstable_call` was successful.
+
+`output` is the hexadecimal-encoded output of the runtime function call.
+
+No more event will be generated with this `operationId`.
+
+### operation-storage-items
+
+```json
+{
+    "event": "operation-storage-items",
+    "operationId": ...,
+    "items": [
+        {
+            "key": "0x0000000...",
+            "value": "0x0000000...",
+            "hash": "0x0000000...",
+            "closest-descendant-merkle-value": "0x000000..."
+        },
+        ...
+    ]
+}
+```
+
+`operationId` is a string returned by `chainHead_unstable_storage`.
+
+Yields one or more items that were found in the storage.
+
+The `key` field is a string containing the hexadecimal-encoded key of the item. This `key` is guaranteed to start with one of the `key`s provided as parameter to `chainHead_unstable_storage`.
+If the `type` parameter was `"value"`, `"hash"`, `"closest-descendant-merkle-value"`, then it is also guaranteed to be equal to one of the `key`s provided as parameter to `chainHead_unstable_storage`.
+
+In the situation where the `type` parameter was `"closest-descendant-merkle-value"`, the fact that `key` is equal to a `key` that was provided as parameter is necessary in order to avoid ambiguities when multiple `items` of type `"closest-descendant-merkle-value"` were requested.
+
+The `value` field is set if this item corresponds to one of the requested items whose `type` was `"value"` or `"descendants-values"`. The `value` field is a string containing the hexadecimal-encoded value of the storage entry.
+
+The `hash` field is set if this item corresponds to one of the requested items whose `type` was `"hash"` or `"descendants-hashes"`. The `hash` field is a string containing the hexadecimal-encoded hash of the storage entry.
+
+The `closest-descendant-merkle-value` field is set if this item corresponds to one of the requested items whose `type` was `"closest-descendant-merkle-value"`. The trie node whose Merkle value is indicated in `closest-descendant-merkle-value` is not indicated, as determining the key of this node might incur an overhead for the JSON-RPC server.
+
+### operation-waiting-for-continue
```

Comment:
For consistency sake and for clarity, shouldn't this be named `operation-storage-waiting-for-continue`? 🤔

Reply:
I've omitted the `storage` because it's starting to be a bit long, and because in principle all operations could generate this event. Of course then I guess `chainHead_storageContinue` should be just `chainHead_continue`.

Reply:
Pushed more changes.

Reply:
> and because in theory all operations could generate this eventt. Only `storage` can right now, but in version 2 maybe other operations could as well.

Ok, that makes sense.

> Of course then I guess `chainHead_storageContinue` should be just `chainHead_continue`.

Yes, please!